framework/src/userobjects/Terminator.C

```diff
@@ -112,12 +116,17 @@ Terminator::handleMessage()
     message = "Terminator '" + name() + "' is causing ";
     if (_fail_mode == FailMode::HARD)
       message += "the execution to terminate.\n";
-    else
+    else if (_fail_mode == FailMode::SOFT)
       message += "a time step cutback by marking the current step as failed.\n";
+    else
+      message = "";
```

Comment:
Not a huge deal, but wouldn't the message sound incomplete in that NONE case? ("Terminator '" + name() + "' is causing ")

Reply:
I ll rework this